packages/eslint-config-typescript/lib/extensions-requiring-type-checking.js

```diff
@@ -7,12 +7,12 @@ module.exports = {
         '@typescript-eslint/no-implied-eval': 'error',
 
         'no-throw-literal': 'off',
-        '@typescript-eslint/no-throw-literal': 'error',
+        '@typescript-eslint/only-throw-error': 'error',
```

Comment:
Can you comment on this rule change? What was the rationale? Is it a behavior change? If it is, it would be good to call it out explicitly in the changefile.

Reply:
Sounds like it has identical functionality so the name change should be innocuous, this seems like a good change.